### ISSUE TITLE
[Android]Blocking elements from being focused

### ISSUE THREAD

**Issue by grgr-dkrk** [react-native-picker/picker#241]
https://github.com/facebook/react-native/issues/30850

`ImportantForAccessibility="no"` and `ImportantForAccessibility="no-hide-descendants"` does not work.
This means that Picker has no control over focus on TalkBack.

I'm in the process of planning a fix for the original Picker in [this PR](https://github.com/facebook/react-native/pull/31296).
I will work on this issue as soon as it is merged.
